app/controllers/instances_controller.rb

```diff
@@ -206,10 +205,16 @@ def typeahead_for_name_showing_references_to_update_instance
 
   def find_instance
     @instance = Instance.find(params[:id])
+    if Rails.configuration.try('foa_profile_aware')
+      # TODO: Confirm with Greg on how we are going to identify a product from the UI
+      @product = Profile::Product.find_by(name: "FOA") 
```

Comment:
@gregclar this is the hardcoded product that I was talking about. Not sure yet where to get this information for a logged in user

Reply:
Yes, leave the TODO there to remind us - although I really don't know the plan for that so it's sth we'll have to find out from Anne eventually .